shared/src/metabase/shared/parameters/parameters.cljc

```diff
@@ -4,51 +4,65 @@
       (:clj
        [(:require [clojure.string :as str]
                   [metabase.mbql.normalize :as mbql.normalize]
-                  [metabase.shared.util.i18n :refer [trs]])]
+                  [metabase.shared.util.i18n :refer [trs]]
+                  [metabase.util.date-2 :as u.date]
+                  [metabase.util.date-2.parse.builder :as b]
+                  [metabase.util.i18n.impl :as i18n.impl])
+        (:import java.time.format.DateTimeFormatter)]
        :cljs
        [(:require ["moment" :as moment]
                   [clojure.string :as str]
                   [metabase.mbql.normalize :as mbql.normalize]
                   [metabase.shared.util.i18n :refer [trs]])]))
 
-(defn- formatted-list
-  [values]
-  (str (str/join ", " (butlast values)) " " (trs "and") " " (last values)))
+#?(:cljs (comment moment/keep-me))
 
 (defmulti formatted-value
   "Formats a value appropriately for inclusion in a text card, based on its type. Does not do any escaping.
   For datetime parameters, the logic here should mirror the logic (as best as possible) in
   frontend/src/metabase/parameters/utils/date-formatting.ts"
-  (fn [tyype _value] (keyword tyype)))
+  (fn [tyype _locale _value] (keyword tyype)))
```

Comment:
Yeah... the sole reason was so that I could use `partial` within the `condp` clauses of the `:date/all-options`. Probably not a good enough justification though, lol. I'll change it.